### PR TITLE
Installing rubocop

### PR DESCRIPTION
**Pull request by carolyncole**
Fixed errors with 'rubocop -a'
Added to circleci

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,22 +38,23 @@ gem 'jbuilder', '~> 2.5'
...
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'rubocop'
 end
 
 group :development do
   # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
```
Comment (hackmastera) [inline Gemfile]: We should delete this comment or move it with the entry

File: Gemfile [hunk 2]
```diff
@@ -36,22 +38,23 @@ gem 'jbuilder', '~> 2.5'
...
-  gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'web-console', '>= 3.3.0'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
```
Comment (hackmastera) [inline Gemfile]: Another comment that should be deleted or moved

**Comment by carolyncole** [comment]
@hackmastera Not certain on Bixby.  This app will not have any samvera components, so I hesitate to add one. That said, it is nice to start with reasonable defaults. @kevinreiss any thoughts on Bixby?

**Comment by carolyncole** [comment]
@hackmastera this now up to date with bixby.

**Comment by hackmastera** [comment]
@carolyncole what do you think of adding
```
Style/StringLiterals:
   EnforcedStyle: double_quotes
```
in .rubocop.yml?

We've really liked having this consistency in figgy.

**Comment by kevinreiss** [comment]
Bixby and enforcing double_quotes sound good to me.

**Pull request merged**